ovos_classifiers/heuristics/numeric.py

```diff
@@ -2097,3 +2097,563 @@ def _initialize_number_data_az(self, short_scale, speech=True):
+        'moitié': 2
+    })
+
+    _EXTRA_SPACE_FR = ""
+
+    _ORDINAL_BASE_FR = {
+        "1.": "premier",
+        "2.": "deuxième",
+        "3.": "troisième",
+        "4.": "quatrième",
+        "5.": "cinquième",
+        "6.": "sixìème",
+        "7.": "septième",
+        "8.": "huitième",
+        "9.": "neuvième",
+        "10.": "dixième",
+        "11.": "onzième",
+        "12.": "douzième",
+        "13.": "treizième",
+        "14.": "quatorzième",
+        "15.": "quinzième",
+        "16.": "seizième",
+        "17.": "dix-septième",
+        "18.": "dix-huitième",
+        "19.": "dix-neuvième",
+        "20.": "vingtième",
+        "21.": "vingt-et-unième",
+        "22.": "vingt-deuxième",
+        "23.": "vingt-troisième",
+        "24.": "vingt-quatrième",
+        "25.": "vingt-cinquième",
+        "26.": "vingt-sixième",
+        "27.": "vingt-septième",
+        "28.": "vingt-huitième",
+        "29.": "vingt-neuvième",
+        "30.": "trentième",
+        "31.": "trentième-et-unième",
+        "32.": "trentième-deuxième",
+        "33.": "trentième-troisième",
+        "34.": "trentième-quatrième",
+        "35.": "trentième-cinquième",
+        "36.": "trentième-sixième",
+        "37.": "trentième-septième",
+        "38.": "trentième-huitième",
+        "39.": "trentième-neuvième",
+        "40.": "quarantième",
+        "41.": "quarantième-et-unième",
+        "42.": "quarantième-deuxième",
+        "43.": "quarantième-troisième",
+        "44.": "quarantième-quatrième",
+        "45.": "quarantième-cinquième",
+        "46.": "quarantième-sixième",
+        "47.": "quarantième-septième",
+        "48.": "quarantième-huitième",
+        "49.": "quarantième-neuvième",
+        "50.": "cinquante",
+        "51.": "cinquante-et-unième",
+        "52.": "cinquante-deuxième",
+        "53.": "cinquante-troisième",
+        "60.": "soixantième",
+        "70.": "soixante-dixième",
+        "80.": "quatre-vingtième",
+        "90.": "quatre-vingt-dixième",
+        "100.": "centième",
+        "1000.": "millième",
+        "1000000.": "millionième"
+        }
+
+    _LONG_SCALE_FR = OrderedDict([
+        (100, 'cent'),
+        (1000, 'mille'),
+        (1000000, 'million'),
+        (1e9, "milliard"),
+        (1e12, 'billion'),
+        (1e15, "billiard"),
+        (1e18, "trillion"),
+        (1e21, "trilliard"),
+        (1e24, "quadrillion"),
+        (1e27, "quadrilliard")
+    ])
+
+    _MULTIPLIER_FR = set(_LONG_SCALE_FR.values())
+
+    _STRING_LONG_SCALE_FR = invert_dict(_LONG_SCALE_FR)
+
+    # ending manipulation
+    for number, item in _LONG_SCALE_FR.items():
+        if int(number) > 1000:
+            if item.endswith('e'):
+                name = item + 'n'
+                _MULTIPLIER_FR.add(name)
+                _STRING_LONG_SCALE_FR[name] = number
+            else:
+                name = item + 'en'
+                _MULTIPLIER_FR.add(name)
+                _STRING_LONG_SCALE_FR[name] = number
```

Comment:
@emphasize some light in there as well would be awesome :D

Reply:
If I understand correctly, I'm not sure that it's required for French as when you have many you always (99%)  have a a `s` at the end.

- 1 million -> many millions
- 1 trilliard -> many trilliards